common/lib/xmodule/xmodule/js/src/video/035_video_accessible_menu.js

```diff
@@ -196,12 +207,13 @@ function () {
                 case KEY.UP:
                     _previousMenuItemLink(this.menuItemsLinks, index).focus();
                     break;
-                // Scroll down  menu, wrapping at the bottom. Keep menu
-                // open.
+                // Scroll down  menu, wrapping at the bottom. Keep menu open.
+
```

Comment:
I didn't write this script, I just updated it. Want me to go through and make these adjustments?